Decred Wallet/Features/Overview/OverviewViewController.swift

```diff
@@ -101,12 +101,20 @@ class OverviewViewController: UIViewController {
         super.viewWillAppear(animated)
         self.navigationController?.navigationBar.isHidden = true
         self.checkWhetherToPromptForSeedBackup()
+        self.updateMultiWalletBalance()
+        self.updateRecentActivity()
     }
     
     func initializeViews() {
         // Set a scroll listener delegate so we can update the nav bar page title text on user scroll.
         self.parentScrollView.delegate = self
+        self.parentScrollView.alwaysBounceVertical = true
         
+        self.refreshControl = UIRefreshControl()
+        self.refreshControl.tintColor = UIColor.lightGray
+        self.refreshControl.addTarget(self, action: #selector(self.refresh), for: .valueChanged)
+        self.parentScrollView.addSubview(self.refreshControl)
```

Comment:
TransactionsHistory page adds the refreshControl to the table view, I think the same should be done here.
```
self.recentTransactionsTableView.addSubview(self.refreshControl)
```

If this works, rename `refreshControl` to `recentTxTableRefreshControl`.